Supervisor dependency is missing

When I follow the setup instructions in your README I am facing the following issue:

``` shell
 $ npm start

> uncloak@1.1.0 start /uncloak
> supervisor server.js

sh: supervisor: command not found

npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE
npm ERR! uncloak@1.1.0 start: `supervisor server.js`
npm ERR! Exit status 127
npm ERR! 
npm ERR! Failed at the uncloak@1.1.0 start script 'supervisor server.js'.
npm ERR! This is most likely a problem with the uncloak package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     supervisor server.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls uncloak
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /uncloak/npm-debug.log
```

After manually running `npm install supervisor` everything is working fine. So I assume the dependency is missing in your package.json.

--- a/package.json
+++ b/package.json
@@ -30,6 +30,7 @@
     "express": "^4.12.0",
     "lodash": "^3.3.1",
     "node-sass": "^2.0.1",
-    "reveal.js": "^3.0.0"
+    "reveal.js": "^3.0.0",
+    "supervisor": "^0.6.0"
   }
 }

